app/models/event.py

```diff
@@ -106,6 +106,7 @@ class Event(SoftDeletionModel):
     can_pay_onsite = db.Column(db.Boolean, default=False)
     can_pay_by_omise = db.Column(db.Boolean, default=False)
     can_pay_by_alipay = db.Column(db.Boolean, default=False)
+    can_pay_by_paytm = db.Column(db.Boolean, default=False, nullable=False)
```

Comment:
`nullable=False` can be skipped

Reply:
No, why?

Reply:
@iamareebjamal Okay wait, you are right. My bad.
Just as a suggestion, should the `can_pay_by_omise` and similar fields also have this condition added to them?

Reply:
Yes